clients/web/src/rest.js

```diff
@@ -171,6 +172,12 @@ let restRequest = function (opts) {
         throw new Error('restRequest requires a "url" argument');
     }
     args.url = `${getApiRoot()}${args.url.substring(0, 1) === '/' ? '' : '/'}${args.url}`;
+    // jQuery's ajax method prefers `method` over `type`.  Since we specify a
+    // default `method`, if the caller supplies a `type`, use it instead.
+    if (args.type) {
```

Comment:
We can.  Or we could remove line 108 and add one line that says `args.method = args.method || args.type || 'GET'` to be explicit about *our* choices.  If we want are trying to use `$.ajax` defaults, then perhaps we shouldn't deprecate the use of `path`, since `$.ajax` hasn't done so.